mvc/views/tag-helpers/authoring/sample/TagHlp/src/AuthoringTagHelpers/TagHelpers/WebsiteInformationTagHelper2.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using AuthoringTagHelpers.Models;
+
+namespace AuthoringTagHelpers.TagHelpers.Copy
+{
+    public class WebsiteInformationTagHelper : TagHelper
+    {
+        public WebsiteContext Info { get; set; }
+
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            output.TagName = "section";
+            output.Content.SetContent(string.Format(
```

Comment:
Hmm this is a pain to read. Update the Process method here and the other copy versions of the tag helper to:

```C#
public class WebsiteInformationTagHelper : TagHelper
{
    public WebsiteContext Info { get; set; }

    public override void Process(TagHelperContext context, TagHelperOutput output)
    {
        output.TagName = "section";
        output.Content.SetContent(
$@"<ul><li><strong>Version:</strong> {Info.Version}</li>
<li><strong>Copyright Year:</strong> {Info.CopyrightYear}</li>
<li><strong>Approved:</strong> {Info.Approved}</li>
<li><strong>Number of tags to show:</strong> {Info.TagsToShow}</li></ul>");
        output.TagMode = TagMode.StartTagAndEndTag;
    }
}
```

Wayy easier to read/see what's going on:
![image](https://cloud.githubusercontent.com/assets/2008729/9677729/53ac5598-5292-11e5-8179-b15ded74bcdd.png)